manifests/m/Microsoft/TeamsTxNDI/2024.8.1.14/Microsoft.TeamsTxNDI.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.TeamsTxNDI
+PackageVersion: 2024.8.1.14
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.17763.0
+InstallerType: msix
+PackageFamilyName: Microsoft.Teams.TxNdi_8wekyb3d8bbwe
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.Teams
Installers:
```

Reply:
Good to know the dependencies are now supported.
Tested on win 10 and it works with the depedant package NOT installed, also work for repeat isntall / update.
![image](https://github.com/microsoft/winget-pkgs/assets/2151317/c150a06b-7a1a-4647-9ca6-7b5d30aca63b)

Thanks for the code review!